### ISSUE TITLE
Add a step for checking file existence (and properties)

### ISSUE THREAD

**Issue by glum23** [rpm-software-management/ci-dnf-stack#323]
Add a step into file_steps.py that will enable to check whether a file exists (and possibly has some properties like permissions, size, etc.).

**Comment by ignatenkobrain** [comment]
not sure, but probably easier would be to use `test` inline, but obviously would like to see some use-cases to find best solution

**Comment by glum23** [comment]
Well, it's always possible to call bash commands directly, however, it imho breaks the purpose of the behavioral framework. I needed it when checking results of dnf download.

**Comment by ignatenkobrain** [comment]
yeah, when talking about dnf-download it is perfectly valid use-case. Mind sending PR for checking that file exists?

**Comment by kkaarreell** [comment]
I propose to implement step like
```
Then the file {filepath} exists
 | tag         | value                     |
 | size        | 12345                   |
 | md5sum | d123e230wf09se |
 | perm      | 755                       |
```
where the table part would be optional.
